src/view.rs

```diff
@@ -272,16 +282,17 @@ impl<'a> TerminalView<'a> {
 
             if is_inverse || is_selected {
                 std::mem::swap(&mut fg, &mut bg);
-            }
 
-            painter.rect_filled(
```

Comment:
This rect rendered the cell background for example if you run `htop` you can't see any background colors for cells without this rect.

below is expected behaviour
<img width="907" alt="Screenshot 2025-01-15 at 21 55 53" src="https://github.com/user-attachments/assets/a8ec4a33-efe3-4089-b55a-8939dbea8b9a" />

below is actual (with this fix)
<img width="994" alt="Screenshot 2025-01-15 at 21 54 25" src="https://github.com/user-attachments/assets/8742b969-1729-4b5e-bb13-b89b125ebc9c" />

Reply:
I tried to fix this problem, but I can't found ratio in default theme

<img width="1223" alt="image" src="https://github.com/user-attachments/assets/75dc9d65-3961-4206-9c73-38a6b14ddc54" />